src/components/interfaces/MOBILE_API.xml

```diff
@@ -772,6 +772,13 @@
             <description> Optional text to label an app menu button (for certain touchscreen platforms).</description>
         </element>
         
+        <element name="navigationText">
+            <description>Navigation text for UpdateTurnList.</description>
+        </element>
+        <element name="notificationText">
```

Comment:
@IGapchuk please insert a blank line to separate these elements

Reply:
Done. Empty line was added.